evdev/events.py

```diff
@@ -169,15 +185,23 @@ class SynEvent(object):
     the multitouch protocol.
     '''
 
-    __slots__ = 'event'
+    __slots__ = 'event', 'keycode'
+
+    def __init__(self, event, allow_known):
```

Comment:
Should be `allow_unknown`, no?

Reply:
whups, yes, fixed in f24f7f48843876c149ebda3f18842ec5db21d5d2